tutorial-code/Startup.cs

```diff
@@ -43,20 +43,9 @@ public void ConfigureServices(IServiceCollection services)
 		}
 
 		// This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-		public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerFactory loggerFactory)
+		public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILoggerFactory loggerFactory)
 		{
-			loggerFactory.AddConsole(Configuration.GetSection("Logging"));
-			loggerFactory.AddDebug();
-
-			if (env.IsDevelopment())
-			{
-				app.UseDeveloperExceptionPage();
-				app.UseBrowserLink();
-			}
-			else
-			{
-				app.UseExceptionHandler("/Home/Error");
-			}
+			app.UseDeveloperExceptionPage();
```

Comment:
Is it intentional that you're adding this for release builds now too?

Reply:
Good callout, I don’t remember why I went this route but it’s worth revisiting.